fix(oxfmt): Ignore unsupported options

leaysgur opened the pull request:
Fixes #15930

Previously `experimentalOperatorPosition` was suggested in `.oxfmtrc` completion. But it is not actually used(not implemented yet).

As a first step, just ignore it. + Report error => #16088

leaysgur commented:
* **#16085** <a href="https://app.graphite.com/github/pr/oxc-project/oxc/16085?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a> 👈 <a href="https://app.graphite.com/github/pr/oxc-project/oxc/16085?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `main`
<hr />

## How to use the Graphite Merge Queue

Add either label to this PR to merge it via the merge queue:
- <kbd>0-merge</kbd> - adds this PR to the back of the merge queue
- <kbd>hotfix</kbd> - for urgent hot fixes, skip the queue and merge this PR next

You must have a Graphite account in order to use the merge queue. Sign up using [this link](https://app.graphite.com/invite/github/oxc-project?ref=merge-queue-instructions-in-stack-comment&prId=16085).

<h2></h2>
<sub>

An organization admin has enabled the [Graphite Merge Queue](https://graphite.dev/docs/graphite-merge-queue) in this repository.

</sub>

<sub>

Please do not merge from GitHub as this will restart CI on PRs being processed by the merge queue.

</sub>

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.
<!-- Current dependencies on/for this PR: -->

Dunqing commented on crates/oxc_formatter/src/service/oxfmtrc.rs:
```diff
@@ -351,13 +337,9 @@ impl Oxfmtrc {
-                OperatorPositionConfig::End => OperatorPosition::End,
-            };
-        }
+        // [Prettier] experimentalTernaries: boolean
```
How about returning an error directly here instead? Then, we should disable generating the schema for this option

leaysgur replied:
After taking another look, I realized it wasn't as bad as I thought. 🙂 => #16088

leaysgur committed:
fix(oxfmt): Ignore unsupported options (#16085)

Fixes #15930

Previously `experimentalOperatorPosition` was suggested in `.oxfmtrc` completion. But it is not actually used(not implemented yet).

As a first step, just ignore it. + Report error => #16088
